homeassistant/components/unifiprotect/button.py

```diff
@@ -73,6 +93,19 @@ class ProtectButtonEntityDescription(
 )
 
 
+async def _remove_adopt_button(
+    hass: HomeAssistant, device: ProtectAdoptableDeviceModel
+) -> None:
+
+    entity_registry = await er.async_get_registry(hass)
+    if device.is_adopted_by_us:
+        entity_id = entity_registry.async_get_entity_id(
+            Platform.BUTTON, DOMAIN, f"{device.mac}_adopt"
+        )
+        if entity_id is not None:
+            entity_registry.async_remove(entity_id)
```

Comment:
```suggestion
    if device.is_adopted_by_us and (entity_id := entity_registry.async_get_entity_id(Platform.BUTTON, DOMAIN, f"{device.mac}_adopt")):
        entity_registry.async_remove(entity_id)
```
Black will complain about this suggestion